package-res/resources/web/pentaho/data/filter/AbstractFilter.js

```diff
@@ -43,29 +42,23 @@ define([
    * * {@link pentaho.data.filter.AbstractTreeFilter}
    */
   var AbstractFilter = Base.extend("pentaho.data.filter.AbstractFilter", /** @lends pentaho.data.filter.AbstractFilter# */{
+
     /**
-     * Outputs a simple object that serializes the operation described by this filter.
-     * The syntax loosely follows the query language of MongoDB.
-     *
-     * @return {Object} Object.
+     * @inheritdoc
```

Comment:
Running ant-js does not show any docs, because you moved the description to an interface.